library/gradle

```diff
@@ -1,57 +1,57 @@
 Maintainers: Keegan Witt <keeganwitt@gmail.com> (@keeganwitt)
 GitRepo: https://github.com/keeganwitt/docker-gradle.git
-GitCommit: ef27237d21e823cc0e075fff946322b5e551e8c4
+GitCommit: 605a08fb025ad34e8e9d5d2391427886e64fc9f7
 
 
 # Hotspot
 
-Tags: 6.8.3-jdk8, 6.8.3-jdk8-hotspot, 6.8-jdk8, 6.8-jdk8-hotspot, jdk8, jdk8-hotspot, 6.8.3-jdk, 6.8.3-jdk-hotspot, 6.8-jdk, 6.8-jdk-hotspot, jdk, jdk-hotspot, 6.8.3, 6.8.3-hotspot, 6.8, 6.8-hotspot, latest, hotspot
+Tags: 7.0.0-jdk8, 7.0.0-jdk8-hotspot, 7.0-jdk8, 7.0-jdk8-hotspot, jdk8, jdk8-hotspot, 7.0.0-jdk, 7.0.0-jdk-hotspot, 7.0-jdk, 7.0-jdk-hotspot, jdk, jdk-hotspot, 7.0.0, 7.0.0-hotspot, 7.0, 7.0-hotspot, latest, hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 Directory: hotspot/jdk8
 
-Tags: 6.8.3-jre8, 6.8.3-jre8-hotspot, 6.8-jre8, 6.8-jre8-hotspot, jre8, jre8-hotspot, 6.8.3-jre, 6.8.3-jre-hotspot, 6.8-jre, 6.8-jre-hotspot, jre, jre-hotspot
+Tags: 7.0.0-jre8, 7.0.0-jre8-hotspot, 7.0-jre8, 7.0-jre8-hotspot, jre8, jre8-hotspot, 7.0.0-jre, 7.0.0-jre-hotspot, 7.0-jre, 7.0-jre-hotspot, jre, jre-hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 Directory: hotspot/jre8
 
-Tags: 6.8.3-jdk11, 6.8.3-jdk11-hotspot, 6.8-jdk11, 6.8-jdk11-hotspot, jdk11, jdk11-hotspot
+Tags: 7.0.0-jdk11, 7.0.0-jdk11-hotspot, 7.0-jdk11, 7.0-jdk11-hotspot, jdk11, jdk11-hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 Directory: hotspot/jdk11
 
-Tags: 6.8.3-jre11, 6.8.3-jre11-hotspot, 6.8-jre11, 6.8-jre11-hotspot, jre11, jre11-hotspot
+Tags: 7.0.0-jre11, 7.0.0-jre11-hotspot, 7.0-jre11, 7.0-jre11-hotspot, jre11, jre11-hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 Directory: hotspot/jre11
 
-Tags: 6.8.3-jdk15, 6.8.3-jdk15-hotspot, 6.8-jdk15, 6.8-jdk15-hotspot, jdk15, jdk15-hotspot
+Tags: 7.0.0-jdk16, 7.0.0-jdk16-hotspot, 7.0-jdk16, 7.0-jdk16-hotspot, jdk16, jdk16-hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-Directory: hotspot/jdk15
+Directory: hotspot/jdk16
 
-Tags: 6.8.3-jre15, 6.8.3-jre15-hotspot, 6.8-jre15, 6.8-jre15-hotspot, jre15, jre15-hotspot
+Tags: 7.0.0-jre16, 7.0.0-jre16-hotspot, 7.0-jre16, 7.0-jre16-hotspot, jre16, jre16-hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-Directory: hotspot/jre15
+Directory: hotspot/jre16
 
 
 # OpenJ9
 
-Tags: 6.8.3-jdk8-openj9, 6.8-jdk8-openj9, jdk8-openj9, 6.8.3-jdk-openj9, 6.8-jdk-openj9, jdk-openj9, 6.8.3-openj9, 6.8-openj9, openj9
+Tags: 7.0.0-jdk8-openj9, 7.0-jdk8-openj9, jdk8-openj9, 7.0.0-jdk-openj9, 7.0-jdk-openj9, jdk-openj9, 7.0.0-openj9, 7.0-openj9, openj9
 Architectures: amd64, ppc64le, s390x
```

Comment:
Apparently, you could add arm64v8 for all openj9 variants: https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/8-jdk-openj9/images/sha256-605695fedbf637d3425a4b8623a9c4554a73aa9d6c50a311fcef5b350e7aa330?context=explore

Reply:
Ah, OK might as well.